config/config.go

```diff
@@ -0,0 +1,21 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+)
+
+type Config struct {
+	ServerURL string `json:"serverURL"`
+	ServerPort int `json:"serverPort"`
+}
+
+func LoadConfigFromPath(path string, config *Config) error {
+	configFile, err := os.Open(path)
+	if err != nil {
+		return err
+	}
+	defer configFile.Close()
+	decoder := json.NewDecoder(configFile)
```

Comment:
Why take the config as an argument? Wouldn't it be simpler to simply return the config value along with the error, so the client doesn't have to instantiate it and pass a pointer?

Reply:
Yes! See [ec3de58a915baa698d7c229c2dccb67f4a9b8c05](https://github.com/cloudfoundry/cf-k8s-api/pull/5/commits/ec3de58a915baa698d7c229c2dccb67f4a9b8c05)